### PR TITLE
files re-arranged and Path fixed of style and javascript

### PR DESCRIPTION
**Pull request by aaryanXcode**
There is a issue on your deployed project of path,I fixed that issue and re-arranged the file for better understanding.
#changed docs folder to src
#inside src there is a html file and a js folder and a css folder 
#for future development we can easily modify code through this file structure
Please go through it,if you find help ful and useful review it and merge and close this PR.

### CODE REVIEW

File: src/index.html
Comment (yuki00yossi) [inline src/index.html:1]: Thanks for fixing the directory structure!
The demo page is published using GitHub Pages, so I need to place the index.html file directly under the root directory or /docs directory to load it.

Regards

**Review by yuki00yossi — CHANGES_REQUESTED** [review]
Thanks for fixing the directory structure!
The demo page is published using GitHub Pages, so I need to place the index.html file directly under the root directory or /docs directory to load it.

Regards

**Comment by aaryanXcode** [comment]
As you said you need to place index.html under root directory,i did n't changed its position,I just renamed docs to src which contain you index.html a/c to your need and styling folder and script folder inside src. Please go through it, If you will merge this PR i think github automatic republish your files , If not kindly check your files and check css,js file connectivity to html,it is not properly linked ,thats why your published project is not working properly.

**Comment by yuki00yossi** [comment]
I've been a little under the weather and I'm sorry for the delay in getting back to you.
Thank you for your commit!

**Pull request merged**